crypto/rc4/asm/rc4-x86_64.pl

```diff
@@ -541,6 +543,7 @@ sub RC4_loop {
 	add	\$12,%rax
 .Ldone:
 	ret
+.cfi_endproc
```

Comment:
Sorry, I am late at the party, but could you add a .size RC4_options,.-RC4_options here as well?

Reply:
There are

```
globl  RC4_options
.type   RC4_options,\@abi-omnipotent
.align  16
RC4_options:
        lea     .Lopts(%rip),%rax
        mov     OPENSSL_ia32cap_P(%rip),%edx
        bt      \$20,%edx
        jc      .L8xchar
        bt      \$30,%edx
        jnc     .Ldone
        add     \$25,%rax
        ret
.L8xchar:
        add     \$12,%rax
.Ldone:
        ret
.align  64
.Lopts:
.asciz  "rc4(8x,int)"
.asciz  "rc4(8x,char)"
.asciz  "rc4(16x,int)"
.asciz  "RC4 for x86_64, CRYPTOGAMS by <appro\@openssl.org>"
.align  64
.size   RC4_options,.-RC4_options
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
ah, ok, that's probably good enough then.